portfolio/src/main/webapp/index.html

```diff
@@ -28,12 +28,21 @@ <h2>Pictures!</h2>
       <p>I was there for a just a couple of days but I had a lot of fun, this is me visiting Marina del Rey's pier.</p>
       <img src="images/beach.jpg" alt="Sunset at sea" class="standard-size image-on-center"/>
       <p>I took this picture on my last day in California, I really enjoyed seeing the waves and the orange color of the sky.</p>
-      <h2>Random Facts about me </h2> 
+      <h2>Random Facts about me</h2> 
       <p>Click this button to see a random fact about me:</p>
       <button onclick="addRandomFact()">Click Here!</button>
-      <div id="fact-container"></div>
+      <div id="fact-container" class="space-between-elements"></div>
       <a href="https://www.linkedin.com/in/susana-perez-juarez/">LinkedIn profile</a>
       <a href="https://github.com/SusanaPerezJ">GitHub profile</a>
+      <form action="/data" method="POST" class="space-between-elements">
+      <p>Leave a comment below!</p>
```

Comment:
nit: you want tap this forward to be consistent

Reply:
All lines between an open tag like `<form>` and its closing tag `</form>` should have extra spaces in front (like what you did for the line after `<body>`)